### ISSUE TITLE
Three dot menu isn't keyboard activatable

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#1479]
On both the dashboard card and within the problem set view, the three dot menu cannot be opened via keyboard navigation.

This should be activated and sub-options navigated based on a11y feedback in prior version

**Comment by rupeshparab** [comment]
Pushed a fix for this

**Comment by abaranowska1** [comment]
@rupeshparab I can open the three dot menu via keyboard on both the dashboard card and within the problem set view, I can do duplicate and delete, but I cannot confirm the warning which is displayed after click delete button with the keyboard
![Peek 2020-10-27 11-21](https://user-images.githubusercontent.com/61964176/97291415-2d349780-184a-11eb-8918-024eeb2dc90c.gif)

**Comment by rupeshparab** [comment]
Pushed a fix for this, have replaced the popover with a modal for trapping focus

**Comment by abaranowska1** [comment]
**Tested with success**
I can confirm the warning which is displayed after clicking the delete button with the keyboard. It is possible to delete the problem set by keaybord. 
![Peek 2020-11-17 15-29](https://user-images.githubusercontent.com/61964176/99402571-c73bad00-28e9-11eb-8a8a-f79c89207065.gif)
